Magento_CustomerCustomAttributes/templates/customer/form/register.phtml

```diff
@@ -616,3 +616,34 @@
         }
     }
 </script>
+
+<script>
+    require([
+        'jquery',
+        'mage/mage'
+    ], function($) {
+        var dataForm = $('#form-validate'),
+            ignore = <?= $_dob->isEnabled() ? '\'input[id$="full"]\'' : 'null' ?>;
+
+            debugger
+
```

Comment:
extra line and debugger ;)

Reply:
hmm I've remove it yesterday in this commit #67654 - Remove debugger